kloppy/domain/services/__init__.py

```diff
@@ -17,16 +17,16 @@ def attacking_direction_from_frame(frame: Frame) -> AttackingDirection:
     """ This method should only be called for the first frame of a """
     avg_x_home = avg(
         [
-            player.x
-            for player in frame.home_team_player_positions.values()
-            if player
+            coordinates.x
+            for key, coordinates in frame.players_coordinates.items()
+            if "home" in key
```

Comment:
I think this should be:

```python
for player, coordinates in frame.players_coordinates.items()
            if player.team.ground == Ground.HOME
```

Reply:
See comment on the latest commit. 

As mentioned in the previous comment of the pull request, except for the EPTS format, the list of players for each team is empty. So till that is done, we need to check if "home" or "away" are on the key of the players_coordinates.

Reply:
Implemented.